src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -77,37 +74,37 @@ type BaseCommentProps = {
 };
 
 // TODO: Replace and rename to Comment component once threaded replies refactor is fully implemented
-const BaseComment = (props: BaseCommentProps) => {
-    const {
-        created_by,
-        created_at,
-        permissions = {},
-        id,
-        isPending = false,
-        isRepliesLoading = false,
-        error,
-        tagged_message = '',
-        translatedTaggedMessage,
-        translations,
-        currentUser,
-        isDisabled,
-        getAvatarUrl,
-        getUserProfileUrl,
-        getMentionWithQuery,
-        hasReplies = false,
-        mentionSelectorContacts,
-        modified_at,
-        onDelete,
-        onEdit,
-        onSelect,
-        onReplyCreate,
-        onShowReplies,
-        onHideReplies,
-        replies = [],
-        repliesTotalCount = 0,
-        status,
-    } = props;
-
+export const BaseComment = ({
```

Comment:
I alphabetized the props. The real diff is only:
~~onEdit~~
onAnnotationEdit,
annotationActivityLink,
onCommentEdit,